services/drupal/web/themes/epa_theme/templates/content/node--full.html.twig

```diff
@@ -45,7 +49,17 @@
   {% set attributes = attributes.setAttribute('lang', node.field_language.value) %}
 {% endif %}
 
+{% set display_sidenav = false %}
+
 {% if sidenav|render|striptags|trim %}
+  {% set display_sidenav = true %}
+{% endif %}
+
+{% if node.field_wide_template.value %}
+  {% set display_sidenav = false %}
+{% endif %}
```

Comment:
Any reason not to combine these two? `{% if sidenav|render|striptags|trim and not node.field_wide_template.value %} // set it to true`?